### DISCUSSION TITLE
ChatGPT Integration

### DISCUSSION THREAD

**Discussion by hillaoui** [spacedriveapp/spacedrive#915] [Feature Request]
Hello,

I want to start by saying that the work you're doing is amazing and remarkable. Thank you for this amazing and changing tool.

My idea is to integrate ChatGPT into the SpaceDriver using OPENAI API key. 
The purpose of this integration is to make the experience very fluid and to make it easier to ask questions about the files and documents I have in my SpaceDrive.

Also, it would be nice to integrate DALLE-E to generate images and save them inside the SpaceDriver.

**Comment by utkubakir** [comment]
Good idea, I don't think we have plans to do this at the moment however we will have extensions so I don't see why wouldn't this be made possible by a community member in the future.
